arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/ImcFragment.java

```diff
@@ -43,6 +45,8 @@ public class ImcFragment implements TraceFragment {
   private boolean mmuIsSet = false;
   private boolean stpIsSet = false;
 
+  private CallFrame childFrame = null;
```

Comment:
This pointer to the child latest child context simplifies us extracting the relevant data later on. The main point being: when the child terminates `it knows` whether it will `SELF_REVERT` but it does not necessarily know its `REVERT_STAMP`.

Currently the arithmetization just copies the `REVERT_STAMP` rather than `SELF_REVERT x REVERT_STAMP` which would make more logical sense since we only care about the `REVERT_STAMP` of the child in case it self reverts.